README.md

```diff
@@ -101,10 +122,80 @@ Whereas traditionally a Java application is built as a single image layer with t
 
 See also [rules_docker](https://github.com/bazelbuild/rules_docker) for a similar existing container image build tool for the [Bazel build system](https://github.com/bazelbuild/bazel).
 
-## Frequently Asked Questions (FAQ)
+## Known Limitations
 
-*TODO: Add more answers.*
+These limitations will be fixed in the future.
+
+* Does not build OCI images
+* Pushing to Docker Hub does not seem to work
+* Cannot build directly to a Docker daemon.
+
+## Frequently Asked Questions (FAQ)
 
 ### But, I'm not a Java developer.
 
-[bazelbuild/rules_docker](https://github.com/bazelbuild/rules_docker)
+See [rules_docker](https://github.com/bazelbuild/rules_docker) for a similar existing container image build tool for the [Bazel build system](https://github.com/bazelbuild/bazel). The tool can build images for languages such as Python, NodeJS, Java, Scala, Groovy, C, Go, Rust, and D.
+
+### Can I use other authentication methods besides a Docker credential helper?
+
+Other authentication methods will be added in our next release (`v0.2.0`).
+
+### Can I define a custom entrypoint?
+
+The plugin attaches a default entrypoint that will run your application automatically.
+
+When running the image, you can override this default entrypoint with your own custom command.
+
+See [`docker run --entrypoint` reference](https://docs.docker.com/engine/reference/run/#entrypoint-default-command-to-execute-at-runtime) for running the image with Docker.
```

Comment:
since we package it up all different, perhaps we should tell them where everything is?

Reply:
Added another Q&A about the filesystem layout.